service-commands/src/setup.js

```diff
@@ -521,6 +527,9 @@ const identityServiceUp = async () => {
   // Start up the docker network `audius_dev`
   await runSetupCommand(Service.NETWORK, SetupCommand.UP)
 
+  // Start up the Solana test validator
+  await runSetupCommand(Service.SOLANA_VALIDATOR, SetupCommand.UP)
```

Comment:
does this have to be sequential before the programs are brought up?

Reply:
idk i just copied what you did haha. i think?

Reply:
oh this is exactly right, nvm!